app/models/settings/campaign.rb

```diff
@@ -0,0 +1,24 @@
+module Settings
+  class Campaign < RailsSettings::Base
+    self.table_name = :settings_campaigns
+
+    # The configuration is cached, change this if you want to force update
+    # the cache, or call Settings::Campaign.clear_cache
+    cache_prefix { "v1" }
+
+    VALID_URL = %r{\A(http|https)://([/|.\w\s-])*.[a-z]{2,5}(:[0-9]{1,5})?(/.*)?\z}.freeze
```

Comment:
Isn't this the same regexp in `Constants::Settings` ?

Reply:
I deliberated for a while where to put this, in the end I settled on a new `UrlValidator` because it makes the most sense and we did custom validators for things like the emoji regex too.